bin/tictactoe

```diff
@@ -1,3 +1,30 @@
 #!/usr/bin/env ruby
-
 require_relative '../config/environment'
+require_all '../lib'
+require_all '../lib/players'
```

Comment:
These lines shouldn't be needed, they should be covered by the `config/environment`.